rct229/rulesets/ashrae9012019/ruleset_functions/get_building_segment_swh_bat.py

```diff
@@ -33,63 +36,61 @@ def get_building_segment_swh_bat(
 
     """
 
-    building_segment = find_exactly_one_with_field_value(
-        "$.buildings[*].building_segments[*]",
-        "id",
-        building_segment_id,
-        rmd,
+    building_segment = find_exactly_one_building_segment(rmd, building_segment_id)
+
+    building_segment_swh_bat = building_segment.get(
+        "service_water_heating_building_area_type", None
```

Comment:
Minor - no need to specify the default as the default is None in Python.

Reply:
Addressed.